src/modules/marketplaces/models/MarketplaceEventsIndexingInput.ts

```diff
@@ -0,0 +1,25 @@
+import { MarketplaceEventsIndexingArgs } from './MarketplaceEventsIndexingArgs';
+
```

Comment:
can you change it to MarketplaceEventsIndexingRequest just to be same name as the others

Reply:
Done